templates/taxi/confirm_delete_car.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <p>Do you want to delete?{{ car }}</p>
```

Comment:
The message 'Do you want to delete?{{ car }}' is missing a space between the question mark and the car variable. Consider adding a space for better readability: 'Do you want to delete {{ car }}?'